pkg/routing/redisrouter.go

```diff
@@ -167,11 +167,12 @@ func (r *RedisRouter) StartParticipantSignal(ctx context.Context, roomName livek
 		Identity: string(pi.Identity),
 		Name:     string(pi.Name),
 		// connection id is to allow the RTC node to identify where to route the message back to
-		ConnectionId:  string(connectionID),
-		Reconnect:     pi.Reconnect,
-		AutoSubscribe: pi.AutoSubscribe,
-		Client:        pi.Client,
-		GrantsJson:    string(claims),
+		ConnectionId:   string(connectionID),
+		Reconnect:      pi.Reconnect,
+		AutoSubscribe:  pi.AutoSubscribe,
+		Client:         pi.Client,
+		GrantsJson:     string(claims),
+		AdaptiveStream: pi.AdaptiveStream,
```

Comment:
this will also need to be hydrated into ParticipantInit when the [message is received](https://github.com/livekit/livekit-server/blob/master/pkg/routing/redisrouter.go#L258).

Reply:
Thank you. Yeah, realised it when making the change for cloud. Fixed in https://github.com/livekit/livekit-server/pull/631/commits/00861e6937c5e5812fecbc3b6fffa518f91022ed